client/my-sites/plans/jetpack-plans/product-store/utils/get-product-icon.ts

```diff
@@ -124,10 +126,18 @@ const PRODUCT_ICON_MAP: Record< string, IconResource > = {
 		regular: JetpackProductIconCreator,
 		light: '',
 	} ),
-	...setProductsIcon( WOOCOMMERCE_PRODUCTS, {
-		regular: WooCommerceProductIcon,
-		light: WooCommerceProductIcon,
-	} ),
+	...setProductsIcon(
+		WOOCOMMERCE_PRODUCTS,
+		isA8CForAgencies()
+			? {
+					regular: WooCommerceLogo,
+					light: WooCommerceLogo,
+			  }
+			: {
+					regular: WooCommerceProductIcon,
+					light: WooCommerceProductIcon,
+			  }
```

Comment:
I believe it is safe to update this for all environments. If I'm not mistaken, we only referenced Woo product icon in A4A currently.

Reply:
No, this is actually being used in Jetpack Cloud as well. But I think we can go ahead and update that as well. So, I removed the check here.